docs/applications/escrow-payment/fixture/client.pem

```diff
@@ -0,0 +1,16 @@
+-----BEGIN CERTIFICATE-----
+MIICjTCCAjKgAwIBAgIUacmufnXYLxwZkSssxVjBePp4zoUwCgYIKoZIzj0EAwIw
+bzELMAkGA1UEBhMCSlAxDjAMBgNVBAgTBVRva3lvMQ4wDAYDVQQHEwVUb2t5bzEf
+MB0GA1UEChMWU2NhbGFyIEludGVybWVkaWF0ZSBDQTEfMB0GA1UEAxMWU2NhbGFy
+IEludGVybWVkaWF0ZSBDQTAeFw0yMTA2MzAwMzA4MDBaFw0yNDA2MjkwMzA4MDBa
+MEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIEwpTb21lLVN0YXRlMSEwHwYDVQQKExhJ
+bnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNC
+AARSNCGmQdwdhqAGialLsvF/H1B0jBbVZbEG1n7PooQSHzGtAickhowU4+0iyRDq
+g90qe7Ex/dPWHexWNbqkjCLfo4HVMIHSMA4GA1UdDwEB/wQEAwIFoDATBgNVHSUE
+DDAKBggrBgEFBQcDAjAMBgNVHRMBAf8EAjAAMB0GA1UdDgQWBBTRioMqi4AZX+dU
+Y1+YAVtyWMtMszAfBgNVHSMEGDAWgBQgvyjyoVxJUr2q5UpJng2NUCjgAjAxBggr
+BgEFBQcBAQQlMCMwIQYIKwYBBQUHMAGGFWh0dHA6Ly9sb2NhbGhvc3Q6ODg4OTAq
+BgNVHR8EIzAhMB+gHaAbhhlodHRwOi8vbG9jYWxob3N0Ojg4ODgvY3JsMAoGCCqG
+SM49BAMCA0kAMEYCIQCYTdQjk6kNJxOWoGUAsXMGhiTWtrPKrZxxvJjkD87/4QIh
+APfY/gpIqfSu71YWIdRDbk2McECIRgHkXTd8ArIKUz8C
+-----END CERTIFICATE-----
```

Comment:
I guess we should have a blank line as EOF.

Reply:
Added